interfaces/cython/cantera/test/test_convert.py

```diff
@@ -64,82 +64,91 @@ def checkKinetics(self, ref, gas, temperatures, pressures, tol=1e-8):
                 self.assertNear(ref_kr[i], gas_kr[i], rtol=tol, msg='kr '+message)
 
     def test_gri30(self):
-        convertMech('../../data/inputs/gri30.inp',
-                    transportFile='../../data/transport/gri30_tran.dat',
-                    outName='gri30_test.cti', quiet=True)
+        p = self.get_test_data_directory()
+        convertMech(pjoin(p, 'gri30.inp'),
+                    transportFile=pjoin(p, 'gri30_tran.dat'),
+                    outName=pjoin(p, 'gri30_test.cti'), quiet=True)
```

Comment:
Good point. I was trying to avoid spraying random output files all over the user's file system. Would it be better to use a temporary directory in the OS's temp storage?